batch/batch/front_end/front_end.py

```diff
@@ -1673,6 +1702,8 @@ async def _commit_update(app: web.Application, batch_id: int, update_id: int, us
 @routes.delete('/api/v1alpha/batches/{batch_id}')
 @rest_billing_project_users_only
 async def delete_batch(request, userdata, batch_id):  # pylint: disable=unused-argument
+    request['batch_identifier'] = str(batch_id)
+    request['batch_operation'] = 'batch_delete_batch'
```

Comment:
I think this would be better in a decorator like `rest_billing_project_users_only` that adds these fields to the request. Should be able to use `f.__name__` to get the name of the endpoint function.